res/css/photon-components.css

```diff
@@ -48,3 +50,30 @@
 .photon-button-ghost:hover:active {
   background-color: var(--grey-90-a20);
 }
+
+/* --- Inputs --- */
+/* See https://design.firefox.com/photon/components/input-fields.html for the spec */
+.photon-input {
+  box-shadow: none; /* remove the default Firefox style */
+  border: 1px solid var(--grey-90-a20);
+  border-radius: 2px;
+  padding: 0 8px;
+  height: 32px;
+}
+
+.photon-input:hover {
+  border-color: var(--grey-90-a30);
+}
+
+.photon-input:invalid:not(:placeholder-shown) {
```

Comment:
This isn't working in IE because `:placeholder-shown` doesn't work in IE. The result is that IE users won't see the red border. A solution that works in IE would need some more JS in the react component, but I could do it if you think this is useful.